rasa/core/tracker_store.py

```diff
@@ -1325,7 +1325,7 @@ def _event_query(
                 )
             )
 
-        return event_query.order_by(self.SQLEvent.timestamp)
+        return event_query.order_by(self.SQLEvent.id)
```

Comment:
I've compared the performance benefits of ordering by ID versus ordering by timestamp, ID in the description of this pull request

Reply:
Is it possible to add a test that reproduces the issue and shows this fixes it?

Reply:
That's good enough, that's what I actually had in mind since the change is in the tracker store module.